Ui Updates

aine1100 opened the pull request:
i made update onThe input has been updated to clearly inform the user that we require the .zip file they downloaded to retrieve the necessary stats. Additionally, the download card has been improved, and there is potential to include more detailed information on it to enhance user understanding and experience.
<img width="1080" height="1920" alt="instagram-wrapped-card (1)" src="https://github.com/user-attachments/assets/8536843c-1667-472f-b21f-0a2cac009991" />
<img width="1919" height="768" alt="image" src="https://github.com/user-attachments/assets/96965deb-46f9-49bb-9e4e-9b0de9e3b801" />

aine1100 committed:
ux handling mainly to make it easy for the user

aine1100 committed:
updated the card to be downloaded

vantezzen commented:
Amazing, thank you! Could you run "npm run lint -- --fix" to remove the lint errors during build?:

```
14:43:09.772 
./src/components/Wrapped/Slides/Roundup.tsx
14:43:09.772 
5:8  Error: 'getShareUrl' is defined but never used.  unused-imports/no-unused-imports
14:43:09.772 
7:8  Error: 'shareImage' is defined but never used.  unused-imports/no-unused-imports
```

aine1100 committed:
fixed the lint errors and updated the sentry to use updated one

aine1100 commented:
**Pull Request Title:**  
chore: fix linting warnings and modernize Sentry configuration for Next.js 15  

**Description**  

**Summary:**  
This PR resolves linting warnings and updates Sentry configuration for Next.js 15 to enhance code quality, performance, and error tracking.  

**Key Changes:**  

1. **Linting & Type Safety:**  
   - Replaced `<img>` with `next/image` in `ShareCard.tsx`.  
   - Renamed `File` import to `FileIcon` in `FileUpload.tsx` to resolve shadowing.  
   - Fixed missing `useEffect` dependencies in `HideForTime.tsx` and `WrappedPlayerComponent.tsx`.  
   - Addressed unused variable warnings with `_` prefix or ESLint suppressions.  

2. **Sentry Modernization:**  
   - Migrated setup to `src/instrumentation.ts` and `src/instrumentation-client.ts`.  
   - Added `onRequestError` hook and `src/app/global-error.tsx` for error handling.  
   - Removed outdated Sentry config files.  

**Verification Results:**  
- Linting and build run without issues.  
- Verified Sentry works on server and client.

snyk-bot committed:
fix: package.json to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-JS-INFLIGHT-6095116
- https://snyk.io/vuln/SNYK-JS-POSTCSS-5926692

aine1100 committed:
Merge pull request #1 from aine1100/snyk-fix-527553ff593bda0a9b0073d692c7265b

[Snyk] Fix for 2 vulnerabilities

aine1100 committed:
++

aine1100 committed:
updated cards

aine1100 committed:
Merge branch 'master' of https://github.com/aine1100/ig-wrapped

aine1100 committed:
updates

vercel[bot] committed:
Install Vercel Web Analytics

## Vercel Web Analytics Installation Complete

Successfully installed and configured Vercel Web Analytics for this Next.js project.

### Changes Made

**1. Package Installation**
- Added `@vercel/analytics` version ^2.0.1 to project dependencies
- Used pnpm (the project's package manager) to install the package
- Updated `package.json` and `pnpm-lock.yaml`

**2. Analytics Component Integration**
- Modified `src/app/layout.tsx` to add Vercel Analytics
- Added import: `import { Analytics } from "@vercel/analytics/next";`
- Added `<Analytics />` component at the end of the body tag (before `</body>`)
- Followed official Vercel documentation for Next.js App Router setup

**3. Files Modified**
- `src/app/layout.tsx` - Added Analytics import and component
- `package.json` - Added @vercel/analytics dependency
- `pnpm-lock.yaml` - Updated with new dependency tree

### Implementation Details

The implementation follows the official Vercel Analytics Quickstart guide (fetched from https://vercel.com/docs/analytics/quickstart):
- Used the Next.js-specific import path: `@vercel/analytics/next`
- Placed the Analytics component in the root layout file as recommended
- Component is positioned after existing analytics scripts (Simple Analytics and Plausible)
- No additional configuration files needed - the component auto-configures on Vercel deployments

### Verification

✅ Build completed successfully with `pnpm run build`
✅ No TypeScript compilation errors
✅ Application builds and generates all routes correctly
✅ Analytics component properly integrated without breaking existing functionality
✅ Lock files updated correctly

### Next Steps

To enable analytics tracking:
1. Deploy this application to Vercel
2. Navigate to your project dashboard on Vercel
3. Go to Analytics section and click "Enable"
4. After deployment, Vercel will automatically start collecting analytics data
5. Verify setup by checking Network tab for requests to `/_vercel/insights/*`

### Notes

- The existing analytics solutions (Simple Analytics and Plausible) remain unchanged
- Vercel Analytics will work alongside these existing solutions
- No dashboard configuration is required in code - enable via Vercel dashboard after deployment
- Analytics will only track when deployed on Vercel (no local development tracking)

Co-authored-by: Vercel <vercel[bot]@users.noreply.github.com>

aine1100 committed:
Merge pull request #9 from aine1100/vercel/install-vercel-web-analytics-1uc0od

Install Vercel Web Analytics

LCSOGthb commented:
When bro, a year already passed

aine1100 commented:
> When bro, a year already passed

yeah i updated it to sync it with the new for 2025 but continuous to be used for other years each year

LCSOGthb commented:
> > When bro, a year already passed
> 
> yeah i updated it to sync it with the new for 2025 but continuous to be used for other years each year

The creator of this website seems to not actively updating this, most of the website's feature are broken

aine1100 commented:
yeah for sure he no longer active

LCSOGthb commented:
> yeah for sure he no longer active

But if you go check his profile you could still see he's updating its own repo

aine1100 commented:
ok so i think he just left it aside but do you have other open source projects we can start or work on for sure

LCSOGthb commented:
> ok so i think he just left it aside but do you have other open source projects we can start or work on for sure

I'm mainly coding websites now, but you can find some open source projects, or maybe ask if someone needs some help on Reddit

aine1100 commented:
yeah i mean websites that we could collaborate and build some cool products

LCSOGthb commented:
> yeah i mean websites that we could collaborate and build some cool products

This will do, but i usually do the coding myself and with some help from AI (if i really can't do it anymore 😂)

aine1100 commented:
yeah i understand we can just linkup and we see if we build something for sure

LCSOGthb commented:
> yeah i understand we can just linkup and we see if we build something for sure

Sure, but i still have plenty of things to do, like my server deployment issue, networking issue. But they only cost tiny amount of my time so it should be quick.

aine1100 commented:
yeah can i get like your discord so that we can keep connecting

vantezzen commented:
Hey there, sorry I kind of forgot about updating here. I wanted to do a detailed review but thats difficult with 10k+ lines changed.
It looks like the builds runs through fine though - if you want, I can merge this!

LCSOGthb commented:
> yeah can i get like your discord so that we can keep connecting



> yeah can i get like your discord so that we can keep connecting

lcsgaming

This might let you add me on discord

LCSOGthb commented:
> Hey there, sorry I kind of forgot about updating here. I wanted to do a detailed review but thats difficult with 10k+ lines changed. It looks like the builds runs through fine though - if you want, I can merge this!

I found some of the funtions on the website has failed, could you fix that?

aine1100 commented:
you can first check and then merge see whats working and merge it
